runtime/metricsview/query.go

```diff
@@ -174,10 +174,11 @@ func (tr *TimeRange) IsZero() bool {
 }
 
 type Expression struct {
-	Name      string     `mapstructure:"name"`
-	Value     any        `mapstructure:"val"`
-	Condition *Condition `mapstructure:"cond"`
-	Subquery  *Subquery  `mapstructure:"subquery"`
+	Name       string     `mapstructure:"name"`
+	Value      any        `mapstructure:"val"`
+	Condition  *Condition `mapstructure:"cond"`
+	Subquery   *Subquery  `mapstructure:"subquery"`
+	Identifier string     `mapstructure:"identifier"`
```

Comment:
Similar comment – I think it's unsafe even to expose `Identifier` in the metrics query API because it can leak underlying fields that are not exposed through dimension/measure expressions.